docs/internal/release.rst

```diff
@@ -257,6 +266,37 @@ Before Running the Release Script
       export GITHUB_TOKEN=ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00
       export READ_THE_DOCS_TOKEN=ff00ff00ff00ff00ff00ff00ff00ff00ff00ff00
 
+Releasing from a Container
+==========================
+
+Performing the release from a container is recommended as it provides a standardized and repeatable environment.
+
+The following script demonstrates how to prepare a container to perform a release:
+
+  .. code-block:: bash
+
+      podman run --rm -ti ubuntu /bin/bash
+      apt update
+      apt install zip git g++ make libssl-dev zlib1g-dev rustc cargo python3 python3-venv vim
+      cpan -T -i Pithub Net::SFTP::Foreign Time::Piece LWP::UserAgent LWP::Protocol::https Net::SSLeay
```

Comment:
Why is `Net::SSLeay` needed?

Reply:
In my container, Pithub fails without it.

Reply:
Do you know if this is a new requirement of PitHub? If it is, we should update the instructions for Perl dependencies.

Reply:
It is not listed as a dependency at https://metacpan.org/pod/Pithub.
So it must be an optional dependency somewhere down the chain.
It was definitely necessary when using the container.

Maybe we mention it as an optional thing that you may need to install?

Reply:
I just deleted my `local::lib` dir and reinstalled the dependencies from scratch and it didn't install `Net::SSLeany`. Running the script with no options is fine. Is it a runtime error when actually trying to use PitHub during the release process?

Part of me wants to know why it's needed, but it's probably okay to go forward with this, especially if it's only required when running in a container.

Reply:
It was definitely a runtime issue.
I don't have an example of the failure but it contained the url and something like "https protocol not supported."

Reply:
I ran some PitHub code and it works without it on my machine so it's not always required.